client/webserver/locales/en-us.go

```diff
@@ -497,4 +497,19 @@ var EnUS = map[string]string{
 	"No":                          "No",
 	"Yes":                         "Yes",
 	"Treasury Keys":               "Treasury Keys",
+	"Enable Privacy":              "Enable Privacy",
+	"Privacy":                     "Privacy",
+	"Enable":                      "Enable",
+	"Disable Privacy":             "Disable Privacy",
+	"Configure Privacy":           "Configure Privacy",
+	"Privacy active":              "Privacy active",
+	"Privacy off":                 "Privacy off",
+	"staking_disabled":            `Ticket purchasing disabled by <span id="extensionModeAppName"></span>.`,
+	"loading privacy status":      "loading privacy status",
+	"mixing_pw_prompt":            "Enter your password to unlock your wallet turn on privacy",
+	"cspp_addr":                   "CSPP Server Address",
+	"cspp_how":                    "CoinShuffle++ creates outputs that cannot be definitively linked to previous on-chain activity. While output amounts are still visible in this form of privacy, the destruction of linkability means your on-chain history can no longer be inferred from publicly available data.",
+	"privacy_intro":               "When privacy is enabled, all of your funds will be sent through an address history obfuscation service using a protocol called CoinShuffle++.",
```

Comment:
Would be helpful to know that each time you receive funds, you'll have to wait for an additional block to get mined (or maybe more?) before you can use your funds.

Also, a comment on the difference between turning off privacy and disabling privacy would be helpful.